### PR TITLE
feat: student-group model

### PR DESCRIPTION
**Pull request by archontis-cpu**
functions:
- __globalLoadStudentGroups__ to load all StudentGroups list;
- __loadStudentGroupsById__ to load concrete group;
- __editStudentGroup__ to edit concrete group;
- __addStudentGroup__ to create a group;

### CODE REVIEW

File: src/models/student-groups/actions.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+function* watchLoadingStudentGroups() {
+  yield takeLatest(actionTypes.LOAD_STUDENT_GROUPS, loadStudentGroupsAsync);
+}
+
+export const loadStudentGroupsById = (id) => ({
+  type: actionTypes.LOAD_STUDENT_GROUP_BY_ID,
+  payload: id,
+});
+
+function* loadStudentGroupsByIdAsync({ payload }) {
```
Comment (MozdolevskyiMaksym) [inline src/models/student-groups/actions.js:32]: No need to create additional method to get a specific group by id. You can take this group from all groups

File: src/models/student-groups/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export * from './actions';
```
Comment (MozdolevskyiMaksym) [inline src/models/student-groups/index.js:1]: `export { } from './actions.js';`
use * in common index.js in folder "models"

**Review by MozdolevskyiMaksym — CHANGES_REQUESTED** [review]
.

File: src/models/student-groups/index.js (hunk 2 above)
Comment (archontis-cpu) [reply src/models/student-groups/index.js:1]: Ok, but what is the difference between

```
export * from './actions';
```

and
```
export {
  globalLoadStudentGroups,
  loadStudentGroupsById,
  editStudentGroup,
  addStudentGroup,
  studentGroupsWatcher,
} from './actions';
``` 
?

File: src/models/student-groups/actions.js (hunk 1 above)
Comment (archontis-cpu) [reply src/models/student-groups/actions.js:32]: Sure, but the model is a representation of the backend API. You can use both the saga effect and pre-loaded store.

File: src/models/student-groups/index.js (hunk 2 above)
Comment (MozdolevskyiMaksym) [reply src/models/student-groups/index.js:1]: No difference)
According to [Sergei](https://github.com/sarhan-azizov) comments. If we want to know, which action we would like to use, we go to the folder of specific model, to index.js and check it.

**Review by Winchestery — APPROVED** [review]

**Review by PavelKundenko — APPROVED** [review]

**Review by MozdolevskyiMaksym — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**